util/find_missing_translations.ts

```diff
@@ -102,7 +102,7 @@ const parseJavascriptFile = (file: string, inputLocales: Lang[]) => {
 
     // If we're not inside an object, keep looking for the start of one.
     const openMatchValue = openMatch?.[1];
-    if (!openMatch || !openMatchValue)
+    if (!openMatch || openMatchValue === undefined || openMatchValue.length === 0)
```

Comment:
```suggestion
    if (!openMatch || openMatchValue === undefined)
```

I think this is a bug that if you have a curly brace with no whitespace before it, we wouldn't find missing translations in that block.  I think we shouldn't early out here, but this is a deliberate change over what was there before.